runtime/gcchk/module.xml

```diff
@@ -78,6 +78,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<library name="omrgcstructs" type="external"/>
 			<library name="j9gcstats"/>
 			<library name="omrgcstats" type="external"/>
+			<library name="omrglue" type="external"/>
```

Comment:
Why is this needed? Why not also for cmake builds?

Reply:
It was needed to fix a link failure I was having with my Aaxon builds on Windows. This is the link failure that was fixed from this change: 

`error LNK2019: unresolved external symbol:
"public: enum GC_ArrayletObjectModelBase::ArrayLayout __cdecl GC_ArrayletObjectModel::getArrayletLayout(struct J9Class *,unsigned __int64,unsigned __int64)" (?getArrayletLayout@GC_ArrayletObjectModel@@QEAA?AW4ArrayLayout@GC_ArrayletObjectModelBase@@PEAUJ9Class@@_K1@Z)
referenced in function:
"private: unsigned __int64 __cdecl GC_CheckEngine::checkJ9Object(struct J9JavaVM *,struct J9Object *,struct J9MM_IterateRegionDescriptor *,unsigned __int64)" (?checkJ9Object@GC_CheckEngine@@AEAA_KPEAUJ9JavaVM@@PEAUJ9Object@@PEAUJ9MM_IterateRegionDescriptor@@_K@Z)`

Reply:
What about the other half of my question: why not also for cmake?

Reply:
It was not required for CMake builds (not entirely sure why, but CMake doesn't have any issue linking `ArrayletObjectModelBase` in `CheckEngine.cpp`

Reply:
I don't see the `omrglue` library in the cmake world. @dnakamura Any theories why not?